tests/e2e/cucumber/steps/ui/resources.ts

```diff
@@ -589,3 +590,17 @@ Then(
     await resourceObject.expectPageNumberNotToBeVisible()
   }
 )
+
+When(
+  '{string} uploads {int} small files in personal space',
+  async function (this: World, stepUser: string, numberOfFiles: number): Promise<void> {
+    const files = createTempResources(numberOfFiles)
+      .map((file) => this.filesEnvironment.getFile({ name: file }))
+      .reverse()
+
+    const { page } = this.actorsEnvironment.getActor({ key: stepUser })
+    const resourceObject = new objects.applicationFiles.Resource({ page })
+
+    await resourceObject.uploadOneAtATime({ resources: files })
```

Comment:
@ScharfViktor What is the expected test flow?
1. upload multiples files `at once`?
2. upload multiple files `one at a time`?

Reply:
I meant uploading multiples files at one. I'm sorry I didn't make it clear right away

this case:

https://github.com/owncloud/web/assets/84779829/c054443e-377b-461d-aeda-4733aeed7044

Reply:
so, `no. 1` it is